integration-tests/contract-permits.spec.ts

```diff
@@ -78,13 +78,13 @@ CONFIGS().forEach(({ lib, rpc, setup, createAddress }) => {
   const Tezos = lib;
   Tezos.setPackerProvider(new MichelCodecPacker());
 
-  describe(`Test of contracts having a permit for tzip-17: ${rpc}`, () => {
+  describe(`Test contract origination having a permit for tzip-17 through contract api: ${rpc}`, () => {
     beforeEach(async (done) => {
       await setup(true);
       done();
     });
 
-    test('Permit can be submitted and set', async (done) => {
+    test('Verify contract.originate for a permit contract', async (done) => {
```

Comment:
the previous title was more descriptive

Reply:
changed to: "Verify a Permit can be submitted and set"